networks/deescord.py

```diff
@@ -130,32 +133,49 @@ class Discord(object):
 
 
     async def retweet(self, message):
+        screen_name = message.content.split('|retweet')[1].strip()
 
-        tooter = message.content.split('|retweet')[1]
-        if not tooter:
+        if not screen_name:
             return await self.say(message.channel, 'Who should I retweet?')
 
-        tooter = tooter.strip()
+        else:
+            try:
+                tooter_profile = (self.application.Twitter._twitter
+                                  .show_user(screen_name=screen_name))
 
-        this_server = message.server
+            except TwythonError as e:
+                error('Twython Error: {}'.format(e))
 
-        conf = self.application.Twitter._twitter_conf
+                return await self.say(message.channel, 'There was a problem searching for the Twitter user with the screen name {}. Is this spelled correctly?')
```

Comment:
I was thinking to remind the user to double check spelling if the twitter user was not found. I will remove this as its not that helpful.